ibm_catalog.json

```diff
@@ -992,6 +992,102 @@
         }
       ],
       "flavors": [
+        {
+          "label": "QuickStart",
+          "name": "quickstart",
```

Comment:
Should we name it `quickstart roks`?

Reply:
Nope - its under the "Red Hat OpenShift Container Platform on VPC landing zone" offering - no need to include it in the flavour too. Also roks is internal name and should not be used publically